workspaces/src/network/server.rs

```diff
@@ -111,22 +148,20 @@ impl Drop for SandboxServer {
             return;
         }
 
+        let rpc_port = self.rpc_port();
         let child = self.process.as_mut().unwrap();
 
         info!(
             target: "workspaces",
-            "Cleaning up sandbox: port={}, pid={}",
-            self.rpc_port,
+            "Cleaning up sandbox: port={:?}, pid={}",
+            rpc_port,
             child.id()
         );
 
         child
             .kill()
             .map_err(|e| format!("Could not cleanup sandbox due to: {:?}", e))
             .unwrap();
-
-        // Unlock the ports just in case they have not been preemptively done.
-        self.unlock_lockfiles().unwrap();
```

Comment:
Was this intentional? This should be safe, no?

Reply:
this was indeed intentional, since I noticed that fs2 actually already drops them on `Drop`s so no need to add it here